### PR TITLE
Add championship and list all in index

### LINKED ISSUE #69: Dates are associated with championships rather than sports leagues
Some sports leagues have multiple championships, so the dates should be associated with a championship rather than the overall league.

### LINKED ISSUE #70: Remove sports league index
Sports league index no longer necessary because of #69

### PR DESCRIPTION
**Pull request by axelclark**
Add championship and list all in index
    * Need dates and info associated with championship not sports league
    * Multiple championships in some leagues
    * Dates will change for subsequent years
    * Nested under fantasy_league to get year
    * Added new table and struct for Championship
Closes #69

Remove sports league index
    * Replaced with championship index
Closes #70

### CODE REVIEW

File: web/models/championship.ex [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    field :championship_at, Ecto.DateTime
+    belongs_to :sports_league, Ex338.SportsLeague
+
+    timestamps()
+  end
+
+  @doc """
+  Builds a changeset based on the `struct` and `params`.
+  """
+  def changeset(struct, params \\ %{}) do
```
Comment (credoci) [inline web/models/championship.ex]: Parameter `struct` has same name as the `Kernel.struct` function.

File: web/models/championship.ex [hunk 2]
```diff
@@ -0,0 +1,35 @@
+defmodule Ex338.Championship do
```
Comment (credoci) [inline web/models/championship.ex]: Modules should have a @moduledoc tag.

File: web/models/championship.ex (hunk 1 above)
Comment (credoci) [inline web/models/championship.ex]: Parameter `struct` has same name as the `Kernel.struct` function.

File: web/models/championship.ex (hunk 2 above)
Comment (axelclark) [inline web/models/championship.ex]: added

File: web/models/championship.ex (hunk 1 above)
Comment (axelclark) [inline web/models/championship.ex]: Fixed

**Pull request merged**